src/translations/en.json

```diff
@@ -41,28 +41,28 @@
         "on": "Wet"
       },
       "gas": {
-        "off": "Clear",
-        "on": "Detected"
+        "off": "Nothing",
```

Comment:
I don't like changing clear to nothing myself, but I'll let others weigh in. I also think just changing Detected to Gas detected is unnecessary extra verbosity. I'd only want to change where we can use more specific words as was done for occupancy.

Reply:
Clear will not get cut off in the label badge.